core/src/main/java/gg/solarmc/loader/data/DataLoader.java

```diff
@@ -24,21 +24,23 @@
 public interface DataLoader<D> {
 
 	/**
-	 * Performs the initial data loader for a new user.
+	 * Performs the data load for a specific user. Implementations should accomodate
```

Comment:
This is so that we do not have to keep all dataloaders loaded on bungee, correct?

Reply:
On velocity, correct